public_html/appeal.php

```diff
@@ -264,9 +264,9 @@
 				if ($_GET['value'] == "adminhold") {
 					$bot = new UTRSBot();
 					$time = date('M d, Y H:i:s', time());
-				    $bot->notifyAdmin($appeal->getBlockingAdmin(), array($appeal->getID(), $time));	
+				    $bot->notifyAdmin($appeal->getCommonName(), array($appeal->getID(), $appeal->getCommonName(), $time));	

```

Comment:
Not that I really give a crap, but you still have redundancy here, where you could just make bot->notifyAdmin() force the admin's name into the template.